### ISSUE TITLE
What release is #3521 merged into? 

### ISSUE THREAD

**Issue by m-standfuss** [open-telemetry/opentelemetry-dotnet#3773]
# Question
Hi, we are trying to use the prometheus exporter option for open telemetry. 

Our code currently looks like:

```
.AddPrometheusExporter(opt =>
{
    opt.StartHttpListener = true;
    opt.HttpListenerPrefixes = new string[] { $"{prometheusPrefix}:{prometheusPort}" };
    opt.ScrapeResponseCacheDurationMilliseconds = 0;
    opt.ScrapeEndpointPath = prometheusPath;
})
```
Which works fine for local development as it defaults to localhost. But when we pushed it out to our environment and attempted to scrape these metrics from another pod in our k8s cluster (with all the networking/ports setup to allow the traffic) it didnt work due to the HttpListenerPrefix being set to localhost. We updated it to include a wildcard but are hitting the issue which I believe is fixed due by the https://github.com/open-telemetry/opentelemetry-dotnet/pull/3521 PR. It shows that commit as being tagged with the rc9.5 release.
![image](https://user-images.githubusercontent.com/52842887/196269814-80df16bc-6786-4fe3-a7f1-79866d58dce4.png)

But also is showing tagged as 1.4.0-beta.2:
![image](https://user-images.githubusercontent.com/52842887/196270062-7e855ff8-8f22-4c54-9194-44aa9a8615b3.png)

Trying to determine which release its in so we can know if we need to implement the workaround or not 
https://github.com/open-telemetry/opentelemetry-dotnet/issues/2840#issuecomment-1072977042

Thanks,
Mike 

Use [Github Discussions](https://github.com/open-telemetry/opentelemetry-dotnet/discussions/new).

**Comment by utpilla** [comment]
@m-standfuss This fix is available starting `1.4.0-alpha.2` version of the `OpenTelemetry.Exporter.Prometheus.HttpListener` package. You could use any package >= `1.4.0-alpha.2`.

**Comment by m-standfuss** [comment]
Thanks!
